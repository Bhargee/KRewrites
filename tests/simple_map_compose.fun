map (fun a -> a) map (fun b -> b)
