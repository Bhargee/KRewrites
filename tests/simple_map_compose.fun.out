<T>
    <k>
        map ((( fun (a -> (a + 1)) )) -| (( fun (b -> (b * 3)) )))
    </k>
    <store>
        .Map
    </store>
    <env>
        .Map
    </env>
</T>
